python/ray/dashboard/subprocesses/module.py

```diff
@@ -47,12 +52,17 @@ class SubprocessModule(abc.ABC):
     def __init__(
         self,
         config: SubprocessModuleConfig,
+        parent_process_pid: int,
     ):
         """
         Initialize current module when DashboardHead loading modules.
         :param dashboard_head: The DashboardHead instance.
         """
         self._config = config
+        self._parent_process_pid = parent_process_pid
+        # Lazy init
+        self._gcs_aio_client = None
+        self._parent_process_death_detection_task = None
```

Comment:
No need to have the parent death detection since the subprocess is started with `daemon=True`.